.pre-commit-config.yaml

```diff
@@ -26,8 +26,8 @@ repos:
     hooks:
     -   id: black
         additional_dependencies: ['click<8.1']
--   repo: https://gitlab.com/pycqa/flake8
-    rev: 3.9.2
+-   repo: https://github.com/pycqa/flake8
+    rev: 5.0.4
```

Comment:
Don't change it here, but FYI, we are shifting towards having all these dependencies defined in the project dependencies.  That moves us towards a single point of dependency definitions, including the versions.

https://github.com/Chia-Network/chia-blockchain/blob/6af4f86171b40554f12126bdfd3c1a548e5fa1b6/.pre-commit-config.yaml#L71-L77
```yaml
-   repo: local
    hooks:
    -   id: flake8
        name: Flake8
        entry: ./activated.py flake8
        language: system
        types: [python]
```

Albeit with the duplication of the `activated.*` scripts which isn't super cool...  `:|`